### PR TITLE
Subtask/as 1030 submission accuracy validation

### PR DESCRIPTION
**Pull request by davehlewis**
## Ticket Number

AS-1030

## Description of change
Added validator to question as per AS-2 instructions

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [X] My commit history in this PR is linear
- [X] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/lpa-questionnaire-web-app/src/controllers/accuracy-submission.js [hunk 1]
```diff
@@ -7,3 +7,26 @@ exports.getAccuracySubmission = (req, res) => {
     backLink: req.session.backLink || `/${req.params.id}/${VIEW.TASK_LIST}`,
   });
 };
+
+exports.postAccuracySubmission = async (req, res) => {
+  const { body } = req;
+  const { errors = {}, errorSummary = [] } = body;
```
Comment (swdpcomputing) [inline packages/lpa-questionnaire-web-app/src/controllers/accuracy-submission.js:13]: errors and errorSummary used once. Do we need a variable?

File: packages/lpa-questionnaire-web-app/tests/unit/validators/accuracy-submission.test.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+      expect(rule.stack[1].validator.name).toEqual('isEmpty');
+      expect(rule.stack[1].negated).toBeTruthy();
+      expect(rule.stack[1].message).toEqual(
+        'Enter details of why this does not accurately reflect the planning application'
+      );
+    });
+  });
+
+  describe('validator', () => {
+    [
```
Comment (swdpcomputing) [inline packages/lpa-questionnaire-web-app/tests/unit/validators/accuracy-submission.test.js:46]: This test could be clearer if large examples are in separate files

Comment (davehlewis) [reply packages/lpa-questionnaire-web-app/tests/unit/validators/accuracy-submission.test.js:46]: Given the shared modular nature of this test, I think it makes sense. As we are doing single questions this structure is ok, if pushing it a little. Plus it keeps us consistent with the other team.

File: packages/lpa-questionnaire-web-app/src/controllers/accuracy-submission.js (hunk 1 above)
Comment (davehlewis) [reply packages/lpa-questionnaire-web-app/src/controllers/accuracy-submission.js:13]: They get used more later, so it's sort of planning ahead, knowing what the final version looks like

**Review by swdpcomputing — APPROVED** [review]

**Pull request merged**